Basic file structure

AlexandreVelloso opened the pull request.

AlexandreVelloso committed:
Basic file structure

fredkneeland commented:
don't delete the current .gitignore... :(  I meant by that issue to put the files in the src/ directory into those folders... sorry for the confusion

AlexandreVelloso committed:
Updated gitignore

Sorry my mess, i am learning english, then i get confused sometimes. I hope that is it

AlexandreVelloso commented:
Sorry my mess, i am learning english, then i get confused sometimes. I hope that is it
